### PR TITLE
Overlays: describe container and position options in more details

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

File: api-reference/10 UI Components/dxLoadPanel/1 Configuration/height.md [hunk 1]
```diff
@@ -11,4 +11,6 @@ default: 90, 60 (Material)
 
 <!-- %fullDescription% -->
 
-<!-- import * from 'api-reference\10 UI Components\DOMComponent\1 Configuration\height.md' -->
\ No newline at end of file
+<!-- import * from 'api-reference\10 UI Components\DOMComponent\1 Configuration\height.md' -->
+
+The {WidgetName} calculates its height relatively to one of the options in the following priority: [container]({basewidgetpath}/Configuration/#container) => [position]({basewidgetpath}/Configuration/#position).[of](/Documentation/ApiReference/Common/Object_Structures/PositionConfig/#of) => **window**.
```
Comment (ksercs) [inline api-reference/10 UI Components/dxLoadPanel/1 Configuration/height.md]: maybe from elements, not from options?

File: api-reference/10 UI Components/dxPopup/1 Configuration/position.md [hunk 2]
```diff
@@ -2,10 +2,21 @@
 id: dxPopup.Options.position
 acceptValues: 'bottom' | 'center' | 'left' | 'left bottom' | 'left top' | 'right' | 'right bottom' | 'right top' | 'top'
 type: String | PositionConfig | function()
+efault: { my: 'center', at: 'center', of: window }
```
Comment (ksercs) [inline api-reference/10 UI Components/dxPopup/1 Configuration/position.md]:
```suggestion
default: { my: 'center', at: 'center', of: window }
```

File: api-reference/10 UI Components/dxPopup/1 Configuration/position.md [hunk 3]
```diff
@@ -2,10 +2,21 @@
...
+firedEvents: positioned
```
Comment (ksercs) [inline api-reference/10 UI Components/dxPopup/1 Configuration/position.md]: this event is private

File: api-reference/10 UI Components/dxLoadPanel/1 Configuration/height.md (hunk 1 above)
Comment (vladaskorohodova) [reply api-reference/10 UI Components/dxLoadPanel/1 Configuration/height.md]: Noted

File: api-reference/10 UI Components/dxPopup/1 Configuration/position.md (hunk 2 above)
Comment (vladaskorohodova) [reply api-reference/10 UI Components/dxPopup/1 Configuration/position.md]: Noted

File: api-reference/10 UI Components/dxLoadPanel/1 Configuration/container.md [hunk 4]
```diff
@@ -8,6 +8,10 @@ default: undefined
 Specifies the UI component's container.
 
 ---
-The default container is defined during the UI component's initialization. It is the viewport, or the body element if the viewport is not found, or the parent element if the previous two are absent.
+The UI component defines the default container on its initialization. This default container can be one of the following (if the element is absent, the component selects the next one):
+- [color swatch container](https://js.devexpress.com/Documentation/Guide/Themes_and_Styles/Predefined_Themes/#Color_Swatches) 
+- **.dx-viewport** 
+- **body**
+
+If you set this property to the particular element, the [shading]({basewidgetpath}/Configuration/#shading) applies to this element. The {WidgetName} calculates its size relatively to this element.
```
Comment (albertov05) [inline api-reference/10 UI Components/dxLoadPanel/1 Configuration/container.md]:
```suggestion
If you set this property to a specific element, the [shading]({basewidgetpath}/Configuration/#shading) applies to this element. The {WidgetName} calculates its size relative to the element.
```

File: api-reference/10 UI Components/dxLoadPanel/1 Configuration/height.md [hunk 5]
```diff
@@ -11,4 +11,6 @@ default: 90, 60 (Material)
 
 <!-- %fullDescription% -->
 
-<!-- import * from 'api-reference\10 UI Components\DOMComponent\1 Configuration\height.md' -->
\ No newline at end of file
+<!-- import * from 'api-reference\10 UI Components\DOMComponent\1 Configuration\height.md' -->
+
+The {WidgetName} calculates its height relatively to one of the elements in the following priority: [container]({basewidgetpath}/Configuration/#container) => [position]({basewidgetpath}/Configuration/#position).[of](/Documentation/ApiReference/Common/Object_Structures/PositionConfig/#of) => **window**.
```
Comment (albertov05) [inline api-reference/10 UI Components/dxLoadPanel/1 Configuration/height.md]:
```suggestion
The {WidgetName} calculates its height relative to one of the elements in the following priority: [container]({basewidgetpath}/Configuration/#container) => [position]({basewidgetpath}/Configuration/#position).[of](/Documentation/ApiReference/Common/Object_Structures/PositionConfig/#of) => **window**.
```

File: api-reference/10 UI Components/dxLoadPanel/1 Configuration/position.md [hunk 6]
```diff
@@ -3,3 +3,5 @@ id: dxLoadPanel.Options.position
 acceptValues: 'bottom' | 'center' | 'left' | 'left bottom' | 'left top' | 'right' | 'right bottom' | 'right top' | 'top'
 type: String | PositionConfig | function()
 ---
+
+If you do not specify the [container]({basewidgetpath}/Configuration/#container) property and set the **position**.[of](/Documentation/ApiReference/Common/Object_Structures/PositionConfig/#of) to the particular element, the [shading]({basewidgetpath}/Configuration/#shading) applies to this element. The {WidgetName} calculates its size relatively to this element.
```
Comment (albertov05) [inline api-reference/10 UI Components/dxLoadPanel/1 Configuration/position.md]:
```suggestion
If you do not specify the [container]({basewidgetpath}/Configuration/#container) property and set the **position**.[of](/Documentation/ApiReference/Common/Object_Structures/PositionConfig/#of) to a specific element, the [shading]({basewidgetpath}/Configuration/#shading) applies to this element. The {WidgetName} calculates its size relative to the element.
```

File: api-reference/10 UI Components/dxLoadPanel/1 Configuration/width.md [hunk 7]
```diff
@@ -11,4 +11,6 @@ default: 222, 60 (Material)
 
 <!-- %fullDescription% -->
 
-<!-- import * from 'api-reference\10 UI Components\DOMComponent\1 Configuration\width.md' -->
\ No newline at end of file
+<!-- import * from 'api-reference\10 UI Components\DOMComponent\1 Configuration\width.md' -->
+
+The {WidgetName} calculates its width relatively to one of the elements in the following priority: [container]({basewidgetpath}/Configuration/#container) => [position]({basewidgetpath}/Configuration/#position).[of](/Documentation/ApiReference/Common/Object_Structures/PositionConfig/#of) => **window**.
```
Comment (albertov05) [inline api-reference/10 UI Components/dxLoadPanel/1 Configuration/width.md]:
```suggestion
The {WidgetName} calculates its width relative to one of the elements in the following priority: [container]({basewidgetpath}/Configuration/#container) => [position]({basewidgetpath}/Configuration/#position).[of](/Documentation/ApiReference/Common/Object_Structures/PositionConfig/#of) => **window**.
```

File: api-reference/10 UI Components/dxPopover/1 Configuration/height.md [hunk 8]
```diff
@@ -11,4 +11,6 @@ Specifies the UI component's height.
 The UI component height.
 
 ---
-#include uiwidgets-ref-height
\ No newline at end of file
+#include uiwidgets-ref-height
+
+The {WidgetName} calculates its height relatively to the **window**.
```
Comment (albertov05) [inline api-reference/10 UI Components/dxPopover/1 Configuration/height.md]:
```suggestion
The {WidgetName} calculates its height relative to the **window**.
```

File: api-reference/10 UI Components/dxPopover/1 Configuration/width.md [hunk 9]
```diff
@@ -11,4 +11,6 @@ Specifies the UI component's width.
 The UI component's width.
 
 ---
-#include uiwidgets-ref-width
\ No newline at end of file
+#include uiwidgets-ref-width
+
+The {WidgetName} calculates its width relatively to the **window**.
```
Comment (albertov05) [inline api-reference/10 UI Components/dxPopover/1 Configuration/width.md]:
```suggestion
The {WidgetName} calculates its width relative to the **window**.
```

File: api-reference/10 UI Components/dxPopup/1 Configuration/container.md [hunk 10]
```diff
@@ -8,9 +8,12 @@ default: undefined
...
 
 ---
-The default container is defined during the UI component's initialization. It is the viewport, the body element if the viewport is not found or the parent element if the previous two are absent.
+The UI component defines the default container on its initialization. This default container can be one of the following (if the element is absent, the component selects the next one):
+- [color swatch container](https://js.devexpress.com/Documentation/Guide/Themes_and_Styles/Predefined_Themes/#Color_Swatches) 
+- **.dx-viewport** 
+- **body**
 
-The specified container affects some features of the {WidgetName}: the area to be shaded, behavior on scrolling, etc.
+If you set this property to the particular element, the [shading]({basewidgetpath}/Configuration/#shading) applies to this element. The {WidgetName} calculates its size relatively to this element.
```
Comment (albertov05) [inline api-reference/10 UI Components/dxPopup/1 Configuration/container.md]:
```suggestion
If you set this property to a specific element, the [shading]({basewidgetpath}/Configuration/#shading) applies to this element. The {WidgetName} calculates its size relative to the element.
```

File: api-reference/10 UI Components/dxPopup/1 Configuration/height.md [hunk 11]
```diff
@@ -11,4 +11,6 @@ firedEvents: resize
 
 <!-- %fullDescription% -->
 
-<!-- import * from 'api-reference\10 UI Components\DOMComponent\1 Configuration\height.md' -->
\ No newline at end of file
+<!-- import * from 'api-reference\10 UI Components\DOMComponent\1 Configuration\height.md' -->
+
+The {WidgetName} calculates its height relatively to one of the elements in the following priority: [container]({basewidgetpath}/Configuration/#container) => [position]({basewidgetpath}/Configuration/#position).[of](/Documentation/ApiReference/Common/Object_Structures/PositionConfig/#of) => **window**.
```
Comment (albertov05) [inline api-reference/10 UI Components/dxPopup/1 Configuration/height.md]:
```suggestion
The {WidgetName} calculates its height relative to one of the elements in the following priority: [container]({basewidgetpath}/Configuration/#container) => [position]({basewidgetpath}/Configuration/#position).[of](/Documentation/ApiReference/Common/Object_Structures/PositionConfig/#of) => **window**.
```

File: api-reference/10 UI Components/dxPopup/1 Configuration/position.md [hunk 12]
```diff
@@ -2,10 +2,20 @@
...
 ---
-<!-- %fullDescription% -->
+---
+##### shortDescription
+Positions the UI component.
+
+---
+#include uiwidgets-ref-position
 
+If you do not specify the [container]({basewidgetpath}/Configuration/#container) property and set the **position**.[of](/Documentation/ApiReference/Common/Object_Structures/PositionConfig/#of) to the particular element, the [shading]({basewidgetpath}/Configuration/#shading) applies to this element. The {WidgetName} calculates its size relatively to this element.
```
Comment (albertov05) [inline api-reference/10 UI Components/dxPopup/1 Configuration/position.md]:
```suggestion
If you do not specify the [container]({basewidgetpath}/Configuration/#container) property and set the **position**.[of](/Documentation/ApiReference/Common/Object_Structures/PositionConfig/#of) to a specific element, the [shading]({basewidgetpath}/Configuration/#shading) applies to this element. The {WidgetName} calculates its size relative to the element.
```

File: api-reference/10 UI Components/dxPopup/1 Configuration/width.md [hunk 13]
```diff
@@ -11,4 +11,6 @@ firedEvents: resize
 
 <!-- %fullDescription% -->
 
-<!-- import * from 'api-reference\10 UI Components\DOMComponent\1 Configuration\width.md' -->
\ No newline at end of file
+<!-- import * from 'api-reference\10 UI Components\DOMComponent\1 Configuration\width.md' -->
+
+The {WidgetName} calculates its width relatively to one of the elements in the following priority: [container]({basewidgetpath}/Configuration/#container) => [position]({basewidgetpath}/Configuration/#position).[of](/Documentation/ApiReference/Common/Object_Structures/PositionConfig/#of) => **window**.
```
Comment (albertov05) [inline api-reference/10 UI Components/dxPopup/1 Configuration/width.md]:
```suggestion
The {WidgetName} calculates its width relative to one of the elements in the following priority: [container]({basewidgetpath}/Configuration/#container) => [position]({basewidgetpath}/Configuration/#position).[of](/Documentation/ApiReference/Common/Object_Structures/PositionConfig/#of) => **window**.
```

File: api-reference/10 UI Components/dxToast/1 Configuration/height.md [hunk 14]
```diff
@@ -11,4 +11,6 @@ default: 'auto'
 
 <!-- %fullDescription% -->
 
-<!-- import * from 'api-reference\10 UI Components\DOMComponent\1 Configuration\height.md' -->
\ No newline at end of file
+<!-- import * from 'api-reference\10 UI Components\DOMComponent\1 Configuration\height.md' -->
+
+The {WidgetName} calculates its height relatively to one of the elements in the following priority: [position]({basewidgetpath}/Configuration/#position).[of](/Documentation/ApiReference/Common/Object_Structures/PositionConfig/#of) => **window**.
```
Comment (albertov05) [inline api-reference/10 UI Components/dxToast/1 Configuration/height.md]:
```suggestion
The {WidgetName} calculates its height relative to one of the elements in the following priority: [position]({basewidgetpath}/Configuration/#position).[of](/Documentation/ApiReference/Common/Object_Structures/PositionConfig/#of) => **window**.
```

File: api-reference/10 UI Components/dxToast/1 Configuration/position.md [hunk 15]
```diff
@@ -3,3 +3,5 @@ id: dxToast.Options.position
 type: PositionConfig | String
 default: 'bottom center', { at: 'bottom left', my: 'bottom left', offset: '20 -20'} (Android), { at: 'bottom center', my: 'bottom center', offset: '0 0' } (phones_on_Android)
 ---
+
+If you set the **position**.[of](/Documentation/ApiReference/Common/Object_Structures/PositionConfig/#of) to the particular element, the [shading]({basewidgetpath}/Configuration/#shading) applies to this element. The {WidgetName} calculates its size relatively to this element.
```
Comment (albertov05) [inline api-reference/10 UI Components/dxToast/1 Configuration/position.md]:
```suggestion
If you set the **position**.[of](/Documentation/ApiReference/Common/Object_Structures/PositionConfig/#of) to a specific element, the [shading]({basewidgetpath}/Configuration/#shading) applies to this element. The {WidgetName} calculates its size relative to the element.
```

File: api-reference/10 UI Components/dxToast/1 Configuration/width.md [hunk 16]
```diff
@@ -11,4 +11,6 @@ default: '80vw', 'auto' (Android), '100vw' (phones_on_Android)
 
 <!-- %fullDescription% -->
 
-<!-- import * from 'api-reference\10 UI Components\DOMComponent\1 Configuration\width.md' -->
\ No newline at end of file
+<!-- import * from 'api-reference\10 UI Components\DOMComponent\1 Configuration\width.md' -->
+
+The {WidgetName} calculates its width relatively to one of the elements in the following priority: [position]({basewidgetpath}/Configuration/#position).[of](/Documentation/ApiReference/Common/Object_Structures/PositionConfig/#of) => **window**.
```
Comment (albertov05) [inline api-reference/10 UI Components/dxToast/1 Configuration/width.md]:
```suggestion
The {WidgetName} calculates its width relative to one of the elements in the following priority: [position]({basewidgetpath}/Configuration/#position).[of](/Documentation/ApiReference/Common/Object_Structures/PositionConfig/#of) => **window**.
```

**Review by ksercs — APPROVED** [review]
LGTM

**Pull request merged**